ISSUE_TEMPLATE.md

```diff
@@ -11,21 +11,21 @@
   1. ...
   1. ...
   1. ...
-  
-  
+
 ## Logs, error output, etc.
- 
- (`epoch.yaml` configuration file, `log/epoch.log` log file plus other related logs, e.g., `log/epoch_sync.log`, `log/epoch_mining.log`; if it’s long, please paste to https://ghostbin.com/ and insert the link here)
- 
- ...
 
+*`epoch.yaml` configuration file*
```

Comment:
`epoch.yaml` 🤔

Reply:
https://www.pivotaltracker.com/story/show/163792443

As I expect we fix in master too, I kept it here as it shall be fixed in both master and minerva branches.

Reply:
epoch.yaml or aeternity.yaml ?

Reply:
https://github.com/aeternity/aeternity/pull/2136